### PR TITLE
AUpdate index.html

### PR DESCRIPTION
**Pull request by yashk0thari**
Added 8 missing speaker profile links....

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -940,6 +940,7 @@ <h1>Speakers</h1>
 											<a href="https://github.com/willhooi"><i class="icon fa fa-github"></i></a>
 											<a href="https://twitter.com/willhooi"><i class="icon social_twitter"></i></a>
 											<a href="https://www.linkedin.com/in/willhooi/"><i class="icon social_linkedin"></i></a>
+                      <a href="https://www.facebook.com/whooi"><i class="icon social_facebook"></i></a>
```
Comment (akshatnitd) [inline index.html]: Fix Indentation!

**Review by akshatnitd — COMMENTED** [review]
Fix Indentation in all links that you have added :)

**Comment by yashk0thari** [comment]
Done!!

**Comment by yashk0thari** [comment]
there are some issues, am closing this PR, will open another...

**Pull request closed without merging**